src/parser/open-api/schemaParser.js

```diff
@@ -116,17 +148,17 @@ export default function getUIReadySchema (jsonSchema) {
     return resolved.oneOf.map(
       (state) => {
         if (state.type === 'array') {
```

Comment:
This block is identical to the other block outside this condition, could they be extracted to a common method?